### DISCUSSION TITLE
Manual uninstall steps

### DISCUSSION THREAD

**Discussion by filippobottega** [valinet/ExplorerPatcher#3501] [Q&A]
Hello,
is it possibile to manual uninstall ExplorerPatcher without using ep_setup.exe?
Currently I have ExplorerPatcher installed but for policies reasons I can't execute ep_setup.exe.
I have administrative rights to edit registry and to delete files in my PC.

Do you have a procedure to do it manually? Do I have to reverse engineering ep_setup.exe?

Thank you for your support,
Filippo

**Comment by Amrsatrio** [comment]
Hi, we do not have manual steps outlined yet, but you can see what's placed/removed by ep_setup by finding for references in this file:
https://github.com/valinet/ExplorerPatcher/blob/master/ep_setup/ep_setup.c
